docs/src/test/java/docs/http/javadsl/server/JsonStreamingExamplesTest.java

```diff
@@ -84,24 +80,31 @@ final Route tweets() {
       )
     );
     //#response-streaming
+    return responseStreaming;
+  }
+
+  final Route measurements() {
+    //#measurement-format
+    final Unmarshaller<ByteString, Measurement> Measurements = Jackson.byteStringUnmarshaller(Measurement.class);
+    //#measurement-format
 
     //#incoming-request-streaming
-    final Route incomingStreaming = path("tweets", () ->
+    final Route incomingStreaming = path("metrics", () ->
```

Comment:
The incoming streaming example should be for `Measurement` not `Tweet`, which is described in the "Consuming JSON Streaming uploads" section.